lib/src/asset_graph/node.dart

```diff
@@ -16,32 +15,56 @@ class AssetNode {
 
   AssetNode(this.id);
 
+  factory AssetNode.deserialize(List serializedNode) {
```

Comment:
Cool - I have experience with getting them set up, so feel free to reach out to chat about them when the time comes.